.gitignore

```diff
@@ -5,4 +5,5 @@
 node_modules
 .DS_Store
 recipes/.prettierrc
-*.war
\ No newline at end of file
+*.war
```

Comment:
@mbruzina are those intended? I don't think we should have .war file I agree, but I'm not sure about the test/definitions exclusion

Reply:
the `test/definitions/apm` line was added because I moved over some of the java apm tests that were migrated to definitions-eu to run locally - I had an incorrect setting in my eu creds and this was faster.  I will remove.